clientutils/src/main/java/eu/europeana/api/commons_sb3/http/HttpConnection.java

```diff
@@ -133,9 +133,31 @@ public HttpResponseHandler post(String url, String requestBody, String contentTy
 		return executeHttpClient(post);
 	}
 
+	/**
+	 * Makes POST request with given url,request, headers and authHandler
+	 * @param url request URL
+	 * @param requestBody body
+	 * @param headers Request headers
+	 * @param auth Authentication handler for the request
+	 * @return HttpResponseHandler that comprises response body as String and status code.
+	 * @throws IOException
+	 */
+	public HttpResponseHandler post(String url, String requestBody,Map<String, String> headers
```

Comment:
Header values can be list, therefore the interface should be updated to use Map<String, Object> for all methods.
If this change will generate compilation errors in other apis, as secodn method can be added

Reply:
to be done later, but I am thinking which header needs multiple values? arent the value` ;` separated??